plugins/statistics.koplugin/main.lua

```diff
@@ -2973,7 +2974,24 @@ function ReaderStatistics:getCurrentBookReadPages()
     return read_pages
 end
 
-function ReaderStatistics.onSync(local_path, cached_path, income_path)
+function ReaderStatistics:isSyncPossible()
```

Comment:
`rgrep Possible` gives nothing, so we're not used to that. May be `ReaderStatistics:canSync()` ?

Reply:
Done :)